client/src/components/AdornedButton.js

```diff
@@ -5,15 +5,21 @@ import Button from "@material-ui/core/Button";
 import CircularProgress from "@material-ui/core/CircularProgress";
 import withStyles from "@material-ui/core/styles/withStyles";
 
-const styles = {
+const styles = theme => ({
     root: {
         marginLeft: 5
+    },
+    spinner: {
+        position: "relative",
+        marginLeft: "7px",
+        marginRight: "5px",
```

Comment:
Could you please remove margin-right, it makes right side of the button with the spinner look wider